docs/arma3/getstarted/steam/README.md

```diff
@@ -1,56 +1,58 @@
-# Using Steam
+# Arma 3 Launcher
```

Comment:
🤔

Reply:
Shouldn't this be "Steam Workshop" or something for consistency? Then it could also be split into Anrop Launcher and Arma 3 Launcher later on